doc/source/myrocks/zenfs.rst

```diff
@@ -11,101 +11,191 @@ which places files into zones on a raw zoned block device (ZBD) using
-#. You can start Percona Server for MySQL with any of the known methods. For example, add this option to ``my.cnf``:
 
-    .. sourcecode:: text
+.. _zenfs-installation:
 
-        [mysqld]
-        ...
-        rocksdb-fs-uri=zenfs://dev:nvme1n2
-        ...
+Installation
+============================================================
 
-You can verify if the ZenFS was successfully created with the following command:
+Start with the installation of *Percona Server for MySQL*. 
 
-.. sourcecode:: bash
+1. The steps are listed here for convenience, for an explanation, see :ref:`apt-install`.
 
-    zenfs ls-uuid
-    ...
-    13e421af-1967-435c-ab15-faf4529710b6    nvme1n2
-    ...
 
-You can check the available storage with the following command:
+   .. code-block:: bash
 
-.. sourcecode:: bash
+      $ sudo apt install gnupg2	
+      $ wget https://repo.percona.com/apt/percona-release_latest.$(lsb_release -sc)_all.deb	
+      $ sudo dpkg -i percona-release_latest.$(lsb_release -sc)_all.deb	
+      $ sudo percona-release setup ps80
 
-    zenfs df --zbd=nvme1n2
-    Free: 7563 MB
-    Used: 0 MB
-    Reclaimable: 0 MB
-    Space amplification: 0%
+2. Install the `zenfs` package. The Percona Server for MySQL with MyRocks and the ZenFS plugin package is listed in the :ref:`installing_from_binary_tarball` section of the *Percona Server for MySQL* installation instructions. 
+   
+   .. code-block:: bash
+	
+      $ sudo apt install percona-server-server
+	
+#. Install the RocksDB plugin package. This package copies ``ha_rocksdb.so`` into a predefined location. **The RocksDB storage engine is not enabled**.
+
+   .. code-block:: bash
+	
+	  $ sudo apt install percona-server-rocksdb
+
+
+.. _zenfs-configure
+
+Configuration
+============================================================
+
+#. Identify your ZBD device, ``<block_device_name>``, with ``lsblk``. Add the ``-o`` option to select which columns are returned. 
+
+   In the example, the ``NAME`` column returns the block device name, the ``SIZE`` column returns the size of the device, and the ``ZONED`` column returns information if the device uses the zone model. The value, ``host-managed``, identifies a ZBD model.
+
+   .. sourcecode:: bash
+
+      lsblk -o NAME,SIZE,ZONED
+      NAME        SIZE  ZONED
+      sda       247.9G  none
+      |-sda1    230.9G  none
+      |-sda2        1G  none
+      |-sda3       16G  none
+      sdb        15.5T  host-managed
+
+		
+#. Change the ownership of ``nvme0n2`` to the ``mysql:mysql`` user account.
+
+   .. code-block:: bash
+	
+	  $ sudo chown mysql:mysql /dev/nvme0n2
+		
+#. Change the permissions so that the user or owner can read and write and the MySQL group can read, in case they must take a backup, for ``nvme0n2``.
+
+   .. code-block:: bash
+	
+	  $ sudo chmod 640 /dev/nvme0n2
+
+#. Change the scheduler for ``nvme0n2`` to ``mq_deadline``.
+
+   .. code-block:: bash
+	
+	  $ sudo bash -c 'echo mq-deadline > /sys/class/block/nvme0n2/queue/scheduler'
```

Comment:
Note that this only changes the I/O scheduler at run time, this configuration will be lost upon the next reboot. I wonder if we should explain on how to configure this in a more permanent fashion.

Reply:
@LenzGr Here
https://unix.stackexchange.com/questions/375600/how-to-enable-and-use-the-bfq-scheduler
I found something similar but for `bfq` scheduler
```
Create /etc/udev/rules.d/60-scheduler.rules if it did not exist and add:

ACTION=="add|change", KERNEL=="sd*[!0-9]|sr*", ATTR{queue/scheduler}="bfq"
```
This is probably the right way to go

Reply:
Added as note